### PR TITLE
Feature/get recipients

### PR DESCRIPTION
**Pull request by escttsalo**
# Description

Makes use of the `dsModel` in the `dsService` folder to allow for api calls to the DS backend to serve the response data in an endpoint for the frontend.

## What work was done?

Added 9 new model functions to `dsModel.js` all returning promises.

- getTotal calls the DS endpoint `/total_served` to retrieve the total recipients served.
- getFamilies calls the DS endpoint `/families_served` to retrieve the number of families served.
- getChildren calls the DS endpoint `/children_served` to retrieve the number of recipients under 18 served, not inclusive.
- getGender calls the DS endpoint `/genders_served` to retrieve the number of genders served.
- getRace calls the DS endpoint `/races_served` to retrieve the number of races served.
- getEthnicities calls the DS endpoint `/ethnicities_served` to retrieve the number of ethnicities served.
- getPrograms calls the DS endpoint `/program_entrollment` to retrieve the number services done for each program.
- getServices calls the DS endpoint `/servies_given` to retrieve the number of services provided.
- getLocations calls the DS endpoint `/locations_of_services` to retrieve the number of services at each location type.

**Note: .env will need to be updated to point `DS_API_URL` to the ds api deployed url.**

Added 9 new endpoints to `metricsRouter.js`, result of a resolved promise is as follows:

- `/total` total number of recipients served.
- `/families` the number of families served.
- `/children` the number of recipients under 18 served.
- `/genders` an array of objects with the number served by gender.
- `/races` an array of objects with the number served by race.
- `/ethnicities` an array of objects with the number served by ethnicity.
- `/programs` an array of objects with the number of services done for each program.
- `/services` an array of objects with the number of services provided.
- `/locations` an array of objects with the number of services at each location type.

## Why was this work done?
BE | As an Administrator, I should be able to view the metric's data on the Reports Tab.
[Trello Card](https://trello.com/c/7nSlrEAN)

## Type of change

- New feature (non-breaking change which adds functionality)?

## Change Status

- Completed waiting on peer review.

## Checklist

- [x] Endpoint tested with Postman and Swagger docs.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

## Video
Original
https://www.loom.com/share/9ea16a9f17614e0787b4ec1ae34b2f2b

Update (10/11/21)
https://www.loom.com/share/bb7901b1bd584aab8a8a540d8c5736d6

### CODE REVIEW

File: api/metrics/metricsRouter.js [hunk 1]
```diff
@@ -1,7 +1,260 @@
 const express = require('express');
 const Metric = require('./metricsModel');
+const DS = require('../dsService/dsModel');
 const router = express.Router();
 
+/**
+ * @swagger
```
Comment (davedwight) [inline api/metrics/metricsRouter.js:7]: nice - awesome that you added swagger for all of these

**Review by praxistechnologies — APPROVED** [review]
Great work with the new endpoints. I like you naming conventions. Makes good sense.

File: api/dsService/dsModel.js [hunk 2]
```diff
@@ -10,4 +10,52 @@ const getViz = (state) => {
   return dsClient.get(`/viz/${state}`);
 };
 
-module.exports = { getPrediction, getViz };
+const getTotal = () => {
+  return dsClient.get('/total_served');
```
Comment (davedwight) [inline api/dsService/dsModel.js:14]: I've not seen axios.create before. What's the reasoning for using this and hitting endpoints in your model functions?

**Review by davedwight — APPROVED** [review]
Looks great! I tested locally with Swagger and it's working as expected :)

Only comment / question would be if you thinks it's cleaner to have the metrics model functions in `metricsModel.js` instead of `dsModel.js` so that the metrics model functions and endpoints are right next to one another.

File: package.json [hunk 3]
```diff
@@ -112,7 +112,7 @@
     "eslint-plugin-standard": "^4.0.1",
     "husky": "^4.3.0",
     "jest": "^25.2.4",
-    "jest-serial-runner": "^1.1.0",
+    "jest-serial-runner": "^1.2.0",
     "lint-staged": "^10.4.2",
     "nodemon": "^2.0.4",
     "prettier": "2.3.2",
```
Comment (michaelquintdev) [inline package.json:118]: I don't think we were supposed to commit the package.json / package-lock.json let me check with Ash quick.

**Review by michaelquintdev — APPROVED** [review]
Checked with ash on the merge with package.json and package-lock.json and should be good to go there. Otherwise endpoints are tested and working from what I've seen.

**Pull request merged**